Public/css/style.css

```diff
@@ -22,6 +22,23 @@ body {
     padding-left: 40px;
 }
 
+/* Sponsors */
+
+.table {
```

Comment:
Is this sponsor specific, if so could use a more specific class

Reply:
Yep good shout